phantomgames/Search/CustomTableViewCell/CustomTableViewCell.swift

```diff
@@ -6,37 +6,52 @@
 //
 
 import UIKit
+import SDWebImage
 
 class CustomTableViewCell: UITableViewCell {
     
     // MARK: IBOutlets
     
     @IBOutlet weak private var icon: UIImageView!
     @IBOutlet weak private var titleLabel: UILabel!
-    @IBOutlet weak private var play: UIButton!
-    @IBOutlet weak private var genreLabel: UILabel!
+    @IBOutlet weak var playButton: UIButton!
+    @IBOutlet weak var genreLabel: UILabel!
+    
+    // MARK: Variables
+    
+    private var gameURL: URL?
     
     // MARK: Functions
     
     override func awakeFromNib() {
         super.awakeFromNib()
+        playButton.addTarget(self, action: #selector(playButtonTapped), for: .touchUpInside)
     }
     
-    func populateWith(game: GamesModel) {
+    func populateWith(game: GameModel) {
         titleLabel.text = game.title
         genreLabel.text = game.genre
+                if let imageURL = URL(string: game.thumbnail) {
+                    icon.sd_setImage(with: imageURL, placeholderImage: UIImage(named: "placeholder"))
+                } else {
+                    icon.image = UIImage(named: "placeholder")
+                }
+             if let gameURL = URL(string: game.gameURL) {
+                 self.gameURL = gameURL
+             }
     }
-    
+     @objc private func playButtonTapped() {
```

Comment:
can we redo this. Please avoid obc stuff at all cost! 
Instead of having the button as an outlet, please create the button as an action. So when you drag and drop from storyboard, you can choose to make it an outlet or an action